sunbird_mcp/src/server.py

```
@@ -0,0 +1,384 @@
+        """
+        try:
+            # Build API URL
+            api_url = f"{settings.API_BASE_URL.rstrip('/')}{CONTENT_ENDPOINT}/{content_id}"
+            
+            # First, get the content details with timeout
+            timeout = aiohttp.ClientTimeout(total=settings.REQUEST_TIMEOUT)
+            async with aiohttp.ClientSession(timeout=timeout) as session:
+                async with session.get(
+                    api_url,
+                    headers={"Content-Type": "application/json"}
+                ) as response:
```

Ruff (0.11.9)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)